build-parent/pom.xml

```diff
@@ -216,9 +216,9 @@
                 <plugin>
                     <artifactId>maven-surefire-plugin</artifactId>
                     <configuration>
-                        <systemProperties>
-                            <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
-                        </systemProperties>
+                        <systemPropertyVariables>
+                          <java.util.logging.manager>org.jboss.logmanager.LogManager</java.util.logging.manager>
```

Comment:
This is minor but you broke the indentation.

Reply:
Fixed